.github/workflows/hardware-smoke.yml

```diff
@@ -44,6 +44,11 @@ jobs:
         with:
           app-path: tt-zephyr-platforms
 
+      - uses: ./tt-zephyr-platforms/.github/workflows/prepare-hardware
+        with:
+          board: ${{ matrix.config.board }}
+          version: '80.15.0.0'
```

Comment:
Might be worth it to bump this to 80.18.0.0, since that includes the regulator fix.